src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1975,7 +1975,6 @@ protected override void AfterMembersCompletedChecks(BindingDiagnosticBag diagnos
                     // The type 'Microsoft.CodeAnalysis.EmbeddedAttribute' must be non-generic, internal, sealed, non-static, have a parameterless constructor, inherit from System.Attribute, and be able to be applied to any type.
                     diagnostics.Add(ErrorCode.ERR_EmbeddedAttributeMustFollowPattern, GetFirstLocation());
                 }
-
```

Comment:
It doesn't look like there are meaningful changes in this file. Consider reverting it to its previous state.